src/shared/packageVersion.ts

```diff
@@ -3,3 +3,4 @@ import { version as packageVersion } from '../../package.json';
 const isProd = process.env.NODE_ENV === 'production';
 
 export const version = isProd ? packageVersion : `${packageVersion}-dev`;
+export const productionVersion = packageVersion;
```

Comment:
Why do we need to clone this variable and not use `packageVersion` everywhere?

Reply:
Well,
1. I want this place to be the only file that accesses `package.json` because it's not common
2. If we export `version` and `packageVersion`, I think it will be hard to understand difference between them. But `productionVersion` at least implies some caution in order to use it.